### PR TITLE
Merge pull request #88 from NoroffFEU/FE-offers

### PR DESCRIPTION
**Pull request by kardokiyani**
New changes to get the header and footer

### CODE REVIEW

**Comment by Christonn93** [comment]
I assisted @kardokiyani with some issues he had, you can see the changes in the last commit.

**Comment by Christonn93** [comment]
@NeNorvalls Did you look over the code? Nothing is missing and everything is looking as it should according to the documentations?

**Comment by NeNorvalls** [comment]
Yeah, I did.  Will finish my part for that today.

File: pages/admin/offers/offer-views/index.html [hunk 1]
```diff
@@ -1,48 +1,45 @@
...
 
-        Bibendum enim facilisis gravida neque convallis a cras. Ut faucibus pulvinar elementum integer enim neque.
-        
-        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
-        
-        Company Name</p>
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (Christonn93) [inline pages/admin/offers/offer-views/index.html:7]: Here its missing the author and team name.

**Pull request merged**